assets/scss/settings/vendor/sweetalert2/_settings.scss

```diff
@@ -0,0 +1,64 @@
+// SweetAlert2
+// -----------------------------------------------------------------------------
+
+
+//
+// Purpose: Settings for the vendor popup box replacement, SweetAlert2
+//
+// -----------------------------------------------------------------------------
+
+/*
+ * Font color, size, family
+ */
+$swal-font-body:                                      stencilFontFamily("body-font"), Arial, Helvetica, sans-serif;
+$swal-font-bodySize:                                  remCalc(stencilNumber("fontSize-root", "px") + 2px);
+$swal-font-bodyColor:                                 stencilColor("color-textBase");
+$swal-font-headings:                                  stencilFontFamily("headings-font"), Arial, Helvetica, sans-serif;
+$swal-font-headingsSize:                              remCalc(stencilNumber("fontSize-h1", "px") + 2px);
+$swal-font-headingsColor:                             stencilColor("color-textHeading");
+
+// Modal stylings
+$swal-modal-bgColor:                                  stencilColor("navUser-dropdown-backgroundColor") !important;
```

Comment:
You should probably introduce a new variable instead of using `navUser-dropdown-backgroundColor`? It looks like `navUser-dropdown-backgroundColor` is intended to used for navigation dropdowns only.